aten/src/ATen/native/xnnpack/RegisterOpContextClass.cpp

```diff
@@ -88,6 +90,7 @@ TORCH_LIBRARY_IMPL(prepacked, CPU, m) {
   m.impl(TORCH_SELECTIVE_NAME("prepacked::conv2d_transpose_clamp_prepack"), TORCH_FN(createConv2dTransposeClampPrePackOpContext));
   m.impl(TORCH_SELECTIVE_NAME("prepacked::conv2d_clamp_run"), TORCH_FN(internal::convolution2d::conv2d_clamp_run));
   m.impl(TORCH_SELECTIVE_NAME("prepacked::conv2d_transpose_clamp_run"), TORCH_FN(internal::convolution2d::conv2d_transpose_clamp_run));
+  m.impl(TORCH_SELECTIVE_NAME("prepacked::hardswish"), TORCH_FN(hardswish));
```

Comment:
We dont need this right? We are just diverting aten::hardswish to this.

Reply:
If you added this so as to test then is there another way to test this? cpp test might be an option but it requires xnnpack::hardswish to be decorated with TORCH_API to change visibility. Maybe just do cpp test but have internal test for this so that we can run it via android test.

Reply:
Moving to internal cpp test